install_config/persistent_storage/persistent-volume-snapshots.adoc

```diff
@@ -0,0 +1,401 @@
+[[persistent-volume-snapshots]]
+= Persistent Volume Snapshots
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Many storage systems provide the ability to create "snapshots" of a persistent
+volume (PV) to protect against data loss. The external snapshot controller and
+provisioner provide means to use the feature in the {product-title} cluster and
+handle volume snapshots through the {product-title} API.
+
+This document describes the current state of volume snapshot support in
+{product-title}. Familiarity with
+link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVs],
+link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[persistent
+volume claims (PVCs)], and
+link:http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html[dynamic
+provisioning] is recommended.
+
+[[features]]
+== Features
+
+* Create snapshot of a `PersistentVolume` bound to a `PersistentVolumeClaim`
+* List existing `VolumeSnapshots`
+* Delete existing `VolumeSnapshot`
+* Create a new `PersistentVolume` from an existing `VolumeSnapshot`
+* Supported `PersistentVolume` link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes[types]:
+** AWS Elastic Block Store (EBS) 
+** Google Compute Engine (GCE) Persistent Disk (PD)
+
+[[installation-setup]]
+== Installation and Setup
+
+The external controller and provisioner are the external components that provide
+volume snapshotting. These external components run in the cluster. The
+controller is responsible for creating, deleting, and reporting events on volume
+snapshots. The provisioner creates new `PersistentVolumes` from the volume
+snapshots. See xref:persistent-volume-snapshots#create-snapshot[Create Snapshot]
+and xref:persistent-volume-snapshots#restore-snapshot[Restore Snapshot] for
+more information.
+
+[[start-controller-and-provisioner]]
+=== Starting the External Controller and Provisioner
+
+The external controller and provisioner services are distributed as container
+images and can be run in the [product-title] cluster as usual. To allow the
```

Comment:
`[product-title]` should be `{product-title}`.

Reply:
Fixed.